lib/node_modules/@stdlib/repl/cli/lib/main.js

```diff
@@ -523,13 +454,28 @@ function REPL( options ) {
 	* @private
 	* @param {string} cmd - command
 	* @param {boolean} success - boolean indicating whether the command successfully executed
+	* @param {*} res - result
+	* @param {boolean} silent - boolean indicating whether the command output should be silenced
 	*/
-	function onCommand( cmd, success ) {
-		self._count += 1;
-		debug( 'Command count: %d', self._count );
+	function onCommand( cmd, success, res, silent ) {
+		var pre;
+		var tmp;
 
-		// Update the internal command history buffer: [..., <id>, <cmd>, <success>, ...]
-		self._history.push( self._count, cmd, success );
+		if ( success === false ) {
+			self._ostream.write( 'Error: '+res.message+'\n' );
+		} else if ( !silent && res !== void 0 ) {
+			pre = replace( self._outputPrompt, '%d', (self._count).toString() );
```

Comment:
```suggestion
			pre = replace( self._outputPrompt, '%d', self._count.toString() );
```

Reply:
`_count` is another magical private variable. This makes it very difficult to reason about class behavior. Yes, `@stdlib/repl/presentation` peers into `@stdlib/repl/lib`, but that doesn't mean it was a good thing or the right decision.